### PR TITLE
[1.0] doc: DVR not supported

### PR DESCRIPTION
**Pull request by csymons-suse**
requested by Product Management & Project Management

### CODE REVIEW

File: doc/source/deployment/requirements.rst [hunk 1]
```diff
@@ -105,5 +101,7 @@ Network Requirements
     the Deployer with dnsmasq to supply DNS functionality, but this is beyond
     the scope of this preview.
 
+Distributed Virtual Routing (DVR) is not supported in this Technology Preview.
```
Comment (jgu17) [inline doc/source/deployment/requirements.rst]: @csymons-suse T.T. asked for a section to state what openstack services are supported, and what features are not supported including dvr. so can we add a section in the overview?

Comment (jgu17) [inline doc/source/deployment/requirements.rst]: "Technical Preview"

Comment (jgu17) [reply doc/source/deployment/requirements.rst]: Also we need to state flat network is not supported. could be a different commit.

Comment (gmingsuse) [reply doc/source/deployment/requirements.rst]: I think the official SUSE term is Technology Preview.

Comment (jgu17) [reply doc/source/deployment/requirements.rst]: addressed in separate PR. so I am good here

Comment (jgu17) [reply doc/source/deployment/requirements.rst]: I added to the home/discaimer page. So all good here

**Comment by dirkmueller** [comment]
unfortunately this was already merged via #518 so this one is no longer relevant (although it has a working commit message)

**Pull request closed without merging**